spec/mailers/mailer_spec.rb

```diff
@@ -17,16 +17,17 @@
       )
     end
 
+    let(:denise) { 'denise@radfordsofsomerford.co.uk' }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.